src/Contrib/Zipkin/SpanConverter.php

```diff
@@ -131,4 +145,127 @@ private static function toSpanKind(SpanDataInterface $span): ?int
         
         return null;
     }
+
+    private static function toAnnotation(EventInterface $event): array
+    {
+        $eventName = $event->getName();
+        $attributesAsJson = SpanConverter::convertEventAttributesToJson($event);
+
+        $value = ($attributesAsJson !== null) ? "\"{$eventName}\": {$attributesAsJson}" : "\"{$eventName}\"";
```

Comment:
You could use ``sprintf('"{%s}"', $eventName)`` to avoid escaping the quotes.

Reply:
Ah nice, that looks much better!